### ISSUE TITLE
As I login as a different user, stumpwm trys to store compiled module files in other user directory.

### ISSUE THREAD

**Issue by kkatsuyuki** [stumpwm/stumpwm#399]
# Problem
I use two users, "guest" and "kiso" and I have used guest for 6 months. 
When I tried to login as "kiso" recentry, an error 
`cannot create directory /home/guest/.config/common-lisp/` is issued and
modules cannot be loaded. On the other hand, when I login as "guest",
modules are loaded correctly and directory `/home/guest/.config/common-lisp/sbcl-1.3.20-linux-x64/`
is created. In this directory, byte-compiled files `cpu.fasl`, `package.fasl`,
etc are created.

How should I edit `~/.stumpwm.d/init.lisp` so that stumpwm can store
byte-compiled files in `/home/kiso/.config/common-lisp/` when
I login as "kiso"?

I don't have much idea about common-lisp and asdf, so please help me.

# my configuration
- use the stumpwm modules `cpu`, `net`, and `pavol`
- `cpu` and `net` modules installed with stumpwm and the path to them is added in default
- I put `pavol` in `/home/kiso/.stumpwm.d/modules/` and add `~/.stumpwm.d/modules/pavol` to load-path.
- kiso's home directory is `/home/kiso/`
- guest's home directory is `/home/guest/`

Followings are in my `~/.stumpwm.d/init.lisp`.

     (add-to-load-path "~/.stumpwm.d/modules/pavol")
     (load-module "pavol)
     (load-module "cpu")
     (load-module "net")

**Comment by dbjergaard** [comment]
I think this is a bug in how stumpwm builds itself. It will take some investigation to determine the correct output.  The issue is that stumpwm dumps an image of sbcl and the corresponding fasl files.  There needs to be some way to tell stumpwm to explicitly load from a directory different from the build path.

Can you try giving the fully qualified path to (add-to-load-path ...) ?

**Comment by kkatsuyuki** [comment]
Sorry for my late reply.

Actually I already resolved this problem. `add-to-load-path` works correctly and stumpwm could load my modules, but the problem is that it tried to strore compliled files in another user (guest's) directory. So I changed output location refering to [asdf Manual](https://common-lisp.net/project/asdf/asdf.html#Controlling-where-ASDF-saves-compiled-files) by

    (asdf:initialize-output-translations
     '(:output-translations (T (:home ".cache" "common-lisp" :implementation))
    :inherit-configuration))
